helper/lang/en.yml

```diff
@@ -510,7 +510,7 @@ en:
           Lists the schedules for a scpecific environment
 
           Example:
-            $ ernest env schedule list
+            $ ernest env schedule list <project> <environment>
```

Comment:
Hey, you may need to also add `$ ernest env schedule list <project> <environment>` to args, as this will be the output used in case you don't provide required arguments.